cli/rt/40_testing.js

```diff
@@ -12,13 +12,19 @@
 
   const disabledConsole = new Console(() => {});
 
+  function maybeColor(fn) {
+    return !globalThis.Deno?.noColor ? fn : (s) => s;
```

Comment:
```suggestion
    return !(globalThis.Deno?.noColor ?? true) ? fn : (s) => s;
```
Maybe this expression should be a function exported from `01_colors.js`.

Reply:
:+1: I'll consider moving this to the color module, for easier access, should it be required in the future